File: Web2 Store/Backend/Store.Core/Models/Article.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Store.Core.Models
{
    public class Article
    {
        [Required]
        [Key]
        public int Id { get; set; }
        [Required]
        [StringLength(100)]
        public string Name { get; set; }
        [Required]
        [StringLength(255)]
        public string Description { get; set; }
        [Required]
        public string Picture { get; set; }
        [Required]
        [Range(0, float.MaxValue)]
        public float Price { get; set; }
        [Required]
        [Range(0, int.MaxValue)]
        public int Quantity { get; set; }
        [Required]
        public int SalesmanId { get; set; }
        public User Salesman { get; set; }
    }
}

File: Web2 Store/frontend/src/models/article.ts
export interface Article{
    id : number;
    name: string;
    price: number;
    quantity: number;
    description: string;
    picture: string;
    salesmanId: number;
   
}

export interface CreateArticle {
    name: string;
    price: number;
    quantity: number;
    description: string;
    picture: string;
    salesmanId: number;
   
}